tests/blinds/test_blindsschedule.py

```diff
@@ -178,12 +181,57 @@ def test_scheduleConflictChecks( self ):
         with pytest.raises( BlindSchedulingException ):
             blindsSchedule.checkHasNoTimeConflicts()
 
+    '''
+    Test for timezone string parsing without error
+    '''
+    def test_validTzParsing( self ): 
+        # Timezone with positive offset
+        tzString = "GMT+0200"
+        expectedTzName = "Etc/GMT+2"
+        parsedTz = BlindsSchedule.tzFromGmtString( tzString )
+        assert( parsedTz.zone == expectedTzName )
+
+        # Timezone with negative offset
+        tzString = "GMT-0600"
+        expectedTzName = "Etc/GMT-6"
+        parsedTz = BlindsSchedule.tzFromGmtString( tzString )
+        assert( parsedTz.zone == expectedTzName )
+
+    '''
+    Test for timezone string parsing with error. This tests highlights unsupported formats
+    '''    
+    def test_invalidTzParsing( self ):
```

Comment:
I don't really want you to change this test but FYI, if you end up having to duplicate code a lot in tests in python you can use the @pytest.mark.parametrize decorator (see: https://docs.pytest.org/en/latest/parametrize.html) where you can add test cases as an array of tuples, so it makes it trivial to implement many test cases with only a few test functions.

Reply:
Good to know. Thanks :)